crates/fluvio-controlplane-metadata/src/smartmodule/spec.rs

```diff
@@ -109,7 +110,7 @@ pub struct SmartModuleWasmSummary {
 pub struct SmartModuleWasm {
     pub format: SmartModuleWasmFormat,
     #[cfg_attr(feature = "use_serde", serde(with = "base64"))]
-    pub payload: Vec<u8>,
+    pub payload: ByteBuf,
```

Comment:
probably we need to bump `fluvio-controlplane-metadata` version

Reply:
I was thinking I could introduce this in a separate PR? Because this PR already has many changes. This way we keep it a bit more progressively.

Reply:
Addressed on: [bd55b27](https://github.com/infinyon/fluvio/pull/2738/commits/bd55b27c6eadc9fd25fe25f90b1b97aca99f552c)